lib/app.rb

```diff
@@ -0,0 +1,26 @@
+module App
+  module_function
```

Comment:
I'm not a big fan of using `module_function` this way:

> Creates module functions for the named methods. These functions may be called with the module as a receiver, and also become available as instance methods to classes that mix in the module.

I generally just use explicit `self.` (or `include self` if lazy) if I don't plan on actually making the module include-able but use it solely as a namespace.

Reply:
Thank you! I'll change that as well!